assets/js/feature-tours/dashboard-sharing-settings.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Dashboard Sharing Settings Feature Tour.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/*
+ * Internal dependencies
+ */
+import { isFeatureEnabled } from '../features';
+
+const { hasMultipleAdmins } = global._googlesitekitUserData || {};
```

Comment:
This state was initially planned as going in the dashboard sharing global but that one loads too late to be usable here without making the user store depend on it, which seems wrong. This state isn't specific to the current user but is user-related, and also the `core/user` store is where feature tours are built into, so from that perspective I think this makes sense.